README.md

```diff
@@ -18,19 +18,11 @@ This repository enables building resin.io for various devices.
 
 `meta-resin` version is kept in `DISTRO_VERSION` variable. `resin-<board>` version is kept in the file called VERSION located in the root of the `resin-<board>` repository and read in the build as variable HOSTOS_VERSION.
 
-There were two version schemes in meta-resin. One that was used before v1.2 and one that is used from 1.2 as it follows:
-
-* Before v1.2
-    * The version of `meta-resin` is in the format of 3 numbers separated by dots.
-    * The version of `resin-<board>` has the same value as the version of `meta-resin` on which it is based on.
-    * Ex. For `meta-resin` release 1.1.1 there can be one `resin-<board>` release 1.1.1.
-* From v1.2
-    * The version of `meta-resin` is in the format is 2 numbers separated by a dot.
-    * The version of `resin-<board>` is in the format of 3 numbers separated by dots. The first two number are the meta-resin release while the last one is the `resin-<board>` release of the respective meta-resin release.
-    * The first `resin-board` release based on a X.Y `meta-resin` release, will be X.Y.0 .
-    * Ex. For `meta-resin` release 1.2 there can be `resin-<board>` releases 1.2.X.
-
-The first versioning scheme was dropped because once meta-resin version A was relased along with the resin-board version A, the resin-board had to wait for a new meta-resin release.
+* The version of `meta-resin` is in the format is 3 numbers separated by a dot. The patch number can have `beta` labels. e.g. 1.2.3, 1.2.3beta1, 2.0.0beta1.
+* The version of `resin-<board>` is constructed by appending to the `meta-resin` version a `rev` label. This will have the semantics of a board revision which adapts a specific `meta-resin` version for a targeted board. For example a meta-resin 1.2.3 can go through 3 board revisions at the end of which the final version will be 1.2.3.rev3.
+* The first `resin-board` release based on `meta-resin` release, will be X.Y.Z.rev0 .
```

Comment:
Maybe rephrase the above to:

"The first `resin-board` release based on `meta-resin` release X.Y.Z, will be X.Y.Z.rev0 ."

for more clarity